### PR TITLE
Workshop

### PR DESCRIPTION
**Pull request by emilyb7**

### CODE REVIEW

**Comment by emilyb7** [comment]
@denesnori i'd be really grateful if you could check this out when you have some time : ) what do you think about asking students to create an access token? I think otherwise rate limiting could be a problem...

I'll publish my solution soon as well, just finishing it off...

**Comment by denesnori** [comment]
I will look at the workshop tonight. :)  I feel it is a bit too early to introduce access tokens on the 3rd week.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <title>GitHub API workshop</title>
+    <link rel="stylesheet" href="./public/styles.css" />
+  </head>
+  <body>
+    <!-- First use API calls to populate the following sections of the DOM with your own data -->
+    <a id="github-link" href="#" target="blank"><h1 id="github-handle">Your name here</h1></a>
+    <img src="#" alt="gitHub-avatar" id="github-avatar" />
+    <p id="repos">Number of repos:&nbsp;<span id="my-repos"></span></p>
+
+    <!-- Now we'll show details of of your repos -->
```
Comment (skibinska) [inline index.html]: small typo - two times *of*

**Comment by skibinska** [comment]
Hi @emilyb7 :) I really like your workshop.

Here are my small suggestions:

- consistent ids/classes naming in index.html (ones are with underscore, others with dashes)
- small typo in this comment - *Now we'll show details of of your repos * (line 14)  - doubles of
- maybe in the README we should mention that we expect waterfall function
- maybe we could add information on **how** to use access token

**Comment by emilyb7** [comment]
@skibinska thanks for the comments, I've made some amendments! : )

**Comment by emilyb7** [comment]
@denesnori @skibinska i've made some final changes.

@skibinska you were right, it was possible to do the whole thing with only one API request! So I've adapted it to make it a bit harder. Hopefully not too overwhelming.

Have also added more tests to the solution branch.

I've assigned to you both. Could one of you merge if you're happy?

**Review by denesnori — APPROVED** [review]

**Pull request merged**